### DISCUSSION TITLE
The influence of the temperature of the model application and the temperature of the training set

### DISCUSSION THREAD

**Discussion by ShangChien** [deepmodeling/deepmd-kit#1316] [General]
If the temperature in the training set is different from the temperature actually used by the model, will this have a greater impact on the simulation results?

**Comment by njzjz** [comment]
Yes. Usually you cannot transfer a model trained by low temperature data into high temperature simulations.

**Comment by xiaoboy185** [comment]
Can i transfer a model trained by high temperature (for example 1500K) data into low temperature (for example 300K ) simulations?
